### ISSUE TITLE
Hide legend

### ISSUE THREAD

**Issue by freevryheid** [dropseed/django-admincharts#31]
Hiding the legend resizes the chart - try this:
```
list_chart_options = {"plugins": {"legend": {"display": 0},},}
```

**Comment by davegaeddert** [comment]
Good point — you would probably want to add `aspectRatio` back in yourself if you override `list_chart_options`:
```python
list_chart_options = {"aspectRatio": 6, "plugins": {"legend": {"display": 0},},}
```

This is the default for `list_chart_options`:
https://github.com/dropseed/django-admincharts/blob/8dd157a1d0ad189668e65d6fd22460b600595ca1/admincharts/admin.py#L9

Let me know if that doesn't answer your question!
